service/controller/resource/pvc/desired_master.go

```diff
@@ -15,30 +16,29 @@ const (
 	EtcdPVSize = "15Gi"
 )
 
-func newEtcdPVCs(customObject v1alpha1.KVMConfig) ([]*corev1.PersistentVolumeClaim, error) {
-	var persistentVolumeClaims []*corev1.PersistentVolumeClaim
+func (r *Resource) getDesiredMasterPVCs(customObject v1alpha1.KVMConfig) ([]corev1.PersistentVolumeClaim, error) {
+	var persistentVolumeClaims []corev1.PersistentVolumeClaim
 
 	for i, masterNode := range customObject.Spec.Cluster.Masters {
 		quantity, err := resource.ParseQuantity(EtcdPVSize)
 		if err != nil {
 			return nil, microerror.Mask(err)
 		}
 
-		persistentVolumeClaim := &corev1.PersistentVolumeClaim{
-			TypeMeta: metav1.TypeMeta{
-				Kind:       "PersistentVolumeClaim",
-				APIVersion: "v1",
-			},
+		persistentVolumeClaim := corev1.PersistentVolumeClaim{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: key.EtcdPVCName(key.ClusterID(customObject), key.VMNumber(i)),
 				Labels: map[string]string{
-					"app":      key.MasterID,
-					"cluster":  key.ClusterID(customObject),
-					"customer": key.ClusterCustomer(customObject),
-					"node":     masterNode.ID,
+					label.ManagedBy:        key.OperatorName,
```

Comment:
Taking this opportunity to use labels from `key` instead of hardcoded ones so they match the labels on the worker PVCs.